code/game/turfs/floor.dm

```diff
@@ -96,6 +96,7 @@
 	else if(is_light_floor())
 		icon_state = "light_broken"
 		broken = 1
+		SetLuminosity(0)
```

Comment:
Does this not have defines

Reply:
I haven't found any, I can create them if needed though

Reply:
that'd be OOS for this PR, luminosity is a pretty big thing